### PR TITLE
chore: update roadmap

### PR DESCRIPTION
**Pull request by marvinlanhenke**
partly #330.

- updated the roadmap to reflect the current status (to the best of my knowledge)

@liurenjie1024 
Can you PTAL at the `reader` section I'm not sure, I could have missed an update here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,19 +50,19 @@ expand to other service.
...
-|------------------------------------------------------------|-------------|
+| ---------------------------------------------------------- | ----------- |
 | File based task planning                                   | Done        |
 | Size based task planning                                   | Not started |
-| Filter pushdown(manifest evaluation, partition prunning)   | Not started |
+| Filter pushdown(manifest evaluation, partition prunning)   | In Progress |
 | Apply deletions, including equality and position deletions | Not started |
-| Read into arrow record batch                               | Not started |
+| Read into arrow record batch                               | Done        |
 | Parquet file support                                       | Not started |
```
Comment (sdd) [inline README.md:59]: We can read parquet files now too.

**Review by sdd — APPROVED** [review]
Thanks! 😁

**Review by liurenjie1024 — COMMENTED** [review]
Thanks @marvinlanhenke for this pr! Let's wait for #314 to be merged

**Review by liurenjie1024 — APPROVED** [review]
Thanks @marvinlanhenke for this pr!

**Comment by liurenjie1024** [comment]
Thanks @marvinlanhenke for this pr, and @sdd for review!

**Pull request merged**